lib/logstash/outputs/syslog.rb

```diff
@@ -94,8 +94,12 @@ class LogStash::Outputs::Syslog < LogStash::Outputs::Base
   # syslog message format: you can choose between rfc3164 or rfc5424
   config :rfc, :validate => ["rfc3164", "rfc5424"], :default => "rfc3164"
 
+  # use codec to format message (instead of setting message)
+  config :usecodec, :validate => :boolean, :default => false
```

Comment:
just use the codec plain, like in https://github.com/logstash-plugins/logstash-input-file/blob/master/lib/logstash/inputs/file.rb#L68-L70 when no codec is need.  what do you think?